pkg/config/net.go

```diff
@@ -27,6 +27,11 @@ func getInterfaceAndNonVIPAddr(vips []net.IP) (vipIface net.Interface, nonVipAdd
 				if _, ok := vipMap[n.IP.String()]; ok {
 					continue // This is a VIP, let's skip
 				}
+				// FIXME: Breaks if more then one interface on the host has the same prefix
```

Comment:
Hmm, is there any way we could look this up? On the bootstrap I think we could get it from the cluster config since that has the machinecidr value, but I'm not sure about the masters. Maybe MCO could dump it out for us?